### PR TITLE
[Master]Let linkdiskbringonline retry

### PR DESCRIPTION
**Pull request by SharpRazor**
linkdiskandbringonline sometimes failed because disks not ready.
So retry it.

### CODE REVIEW

File: smtLayer/vmUtils.py [hunk 1]
```diff
@@ -312,10 +312,27 @@ def installFS(rh, vaddr, mode, fileSystem, diskType):
     strCmd = ' '.join(cmd)
     rh.printSysLog("Invoking: " + strCmd)
     try:
-        out = subprocess.check_output(cmd, close_fds=True)
+        # Sometimes the disk is not ready: sleep and retry
```
Comment (jichenjc) [inline smtLayer/vmUtils.py:315]: yes, maybe you can give  a try on this
thanks a lot for your help

**Comment by jichenjc** [comment]
please fix conflict then let's merge this and backport
then let's see what's happen in 72 hour run

**Comment by SharpRazor** [comment]
> please fix conflict then let's merge this and backport
> then let's see what's happen in 72 hour run

done.

**Comment by jichenjc** [comment]
please backport

**Pull request merged**